docs/governance/decred-constitution.md

```diff
@@ -1,48 +1,41 @@
 - Stakeholder votes recorded in the blockchain are rewarded with 6% of each block subsidy, and each block can have up to 5 votes for a total of 30% of each block subsidy.
 - PoW receives 60% of each block subsidy, subject to the constraint that their subsidy scales linearly with the number of PoS votes included, e.g. including 3 of 5 votes reduces PoW subsidy to 60% of the maximum.
-- The votes themselves decide by majority decision whether the regular transaction tree of the previous block, including the PoW subsidy, is valid. Thus, if PoS voters vote against a particular PoW block, it destroys the PoW reward (and Treasury reward) and invalidates any regular transactions within that block.
-- Additional vote bits may be set when stakeholders submit votes, allowing stakeholders to vote on matters besides the previous block. The method of adopting changes to the consensus rules uses these additional vote bits.
+- The votes themselves decide by majority decision whether the regular transaction tree of the previous block, including the PoW subsidy, is valid. Thus, if PoS voters vote against a particular PoW block, it invalidates the PoW reward (and Treasury reward) and invalidates any regular transactions within that block.
+- Decred's process for [amending the consensus rules](https://docs.decred.org/governance/consensus-rule-voting/consensus-rules-voting/) is also driven by stakeholder voting. The process begins when at least 95% of PoW miners and 75% of PoS voters have upgraded their software to a new version with latent changes to the rules. Once these criteria are met, a voting period of 8,064 blocks (~4 weeks) begins, to decide whether the latent rule changes should be activated. 
+- For a rule change proposal to be approved, at least 75% of the tickets not set to Abstain must vote Yes. If this requirement is met, and a quorum of 10% tickets voting Yes or No is achieved, then the rule change will be activated 8,064 blocks (~4 weeks) later.
 
 ------
 
 ## Project Governance and Funding
 
 * Sustainability and longevity require that 10% of all block rewards be given to a project Treasury fund, administered by a development organization.
-* The initial development organization is *Decred Holdings Group* LLC ("DHG"), a Nevis LLC that is responsible for funding work related to the development of the project, such as software development, infrastructure, and awareness.
-* DHG shall only fund work that adheres to the guiding principles.
+* The initial development organization is *Decred Holdings Group* LLC ("DHG"), a Nevis LLC that is responsible for funding work related to the development of the project, such as software development, infrastructure, and awareness. DHG shall comply with all relevant bodies of law in the jurisdictions where applicable, such as embargoes and other trade sanctions.
 * The project aims to transition to a Decentralized Autonomous Entity which controls the disbursement of Treasury funds.
```

Comment:
I would recommend defining "Treasury" and "Decentralized Autonomous Entity."

Especially "Decentralized Autonomous Entity" because a legal entity is autonomous by its very nature. I am assuming the main difference here is "decentralized," which may require defining "decentralized" in this context.

Reply:
I think that would go way out of the scope I set myself, which was to remove outdated information and add important new information. The Treasury and DAE concept are still very much developing and will have their own proposals to establish key aspects of their definition and how they operate. If someone wants to have a go at defining these terms now they can do so in a follow-up amendment proposal, I would wait until they have reached a more stable form.

Reply:
I see now that they are still left undefined and under deliberation. 

Thank you for the clarification.